### PR TITLE
Update doc PC8800 

### PR DESCRIPTION
**Pull request by FollyMaddy**
I would like to add information to this document

m3u extension added
added info about extensions against missunderstanding (.cmt is a good extension and is not a .cmd) 
added overall improvements discovered in the old-japanese-computer-appreciation-thread

making this pull request directly in github was kind of a struggle, but I'm learning
I hope it works

### CODE REVIEW

File: docs/PC-8800.md [hunk 1]
```diff
@@ -44,40 +25,171 @@ At this link you can find information on the BIOS files of the emulator:
...
 | N88EXT3.ROM | EFCE0B51CAB9F0DA6CF68507757F1245A2867A72 | 7713C519 | 8 KB |
 
-If you add these 2 files too, you will have the translation of the menus in kanji characters:
+Adding the next 2 files :
+
+Menus (Quasi88 only) and Japanese games will display the Japanese kanji characters. [Quasi88, lr-quasi88]
+
+Not adding the next 2 files :
+
+Menus will be in English (Quasi88 only) and Japanese games will **not** display the Japanese kanji characters. [Quasi88, lr-quasi88]
```
Comment (cmitu) [inline docs/PC-8800.md:34]: This section is superfluous, just specify what the BIOS files are for - allow the display of Japanese Kanji characters in the emulator.

File: docs/PC-8800.md [hunk 2]
```diff
@@ -44,40 +25,171 @@ At this link you can find information on the BIOS files of the emulator:
...
 
 | File | SHA1 | CRC32 | Size |
 | :--: | :--: | :--: | :--: |
 | N88KNJ1.ROM | 82E11A177AF6A5091DD67F50A2F4BAFDA84D6556 | 6178BD43 | 128 KB |
 | N88KNJ2.ROM | 7E6591CD465CBB35D6D3446C5A83B46D30FAFE95 | 154803CC | 128 KB |
 
-Put the bios files in lowercase otherwise the emulator won't find them, and result in black screen.
```
Comment (cmitu) [inline docs/PC-8800.md:54]: Why was this deleted - is the file casing not important ?

File: docs/PC-8800.md [hunk 3]
```diff
@@ -44,40 +25,171 @@ At this link you can find information on the BIOS files of the emulator:
...
 ```
 
 Once this is done, the controls will be automatically mapped as such:
 ```
 Left Joystick - Number Pad
-A Button - Z key
-B button - X key
+A Button - "Z key"
+B button - "X key"
+```
```
Comment (cmitu) [inline docs/PC-8800.md:86]: Please use a simple list to show the controls:
* *A* button - Z key
...

It's also not clear what side is the RetroPad and what is the machine input - does the A button on the gamepad simulates the Z key in the emulated machine or vice-verse ?

File: docs/PC-8800.md [hunk 4]
```diff
@@ -44,40 +25,171 @@ At this link you can find information on the BIOS files of the emulator:
...
+Press "Z-key"
+Select, with "X-key", "Subsystems"
+Select, with "X-key", "Load 2-Disk Game"
+Select, with "X-key", the first disk file
+Select, with "X-key", "Subsystems"
+Select, with "X-key", "Load 2-Disk Game"
+Select, with "X-key", the second disk file
+Select, with "X-key", "Subsystems"
+Select, with "X-key", "Start 2-Disk Game"
+```
```
Comment (cmitu) [inline docs/PC-8800.md:144]: Use a simple list to outline the steps, but don't use the keyboard shortcuts for operation - use the RetroPad conventions:
* Open the RGUI with _Hotkey + X_
* Go back (B button)
* Open the _Subsystems_ menu ...
..etc

File: docs/PC-8800.md [hunk 5]
```diff
@@ -44,40 +25,171 @@ At this link you can find information on the BIOS files of the emulator:
...
+Ys II (User disk).d88
+```
+
+## Cycling between disks [lr-quasi88]:
+If you've loaded multiple disks, you can hold one of the shoulder buttons and use the D-Pad(Arrow-keys) to change the disk that's loaded in each drive. 
+Use L1-shoulder(Q-key) for Drive 1 and R1-shoulder(W-key) for Drive 2. 
+When the shoulder button is released, the chosen disk will be inserted.
+
+## Settings for games [lr-quasi88]:
+[Fire Hawk] and many other games only work if the sub_cpu_mode is always using both CPUs.
```
Comment (cmitu) [inline docs/PC-8800.md:162]: Remove the `[..]` and use `**` for emphasis (Bold) for game names. Maybe it's better to have a list of game names. 
Maybe re-phrase as: 
> and many other games work only if the `sub_cpu_mode` option is set to `2` (both CPUs).

File: docs/PC-8800.md [hunk 6]
```diff
@@ -44,40 +25,171 @@ At this link you can find information on the BIOS files of the emulator:
...
+Earlier binary emulator versions didn't have this core option.
+So if it doesn't exist in the core options, you have to install the emulator from source.
+You can set this from the RetroArch menu :
+- Press "F1"
+- Goto "Options"
+- Goto "Sub-CPU Mode"
+- Select "Always run both CPUs"
+- Press "F1" 
+- Press "ESC" (your setting are stored)
+
```
Comment (cmitu) [inline docs/PC-8800.md:173]: Same comment here - don't use the keyboard bindings, but the RetroPad equivalent - _Hotkey + X_, _B_, _A_,etc.
Small typo - _setting_ should be _settings_.

File: docs/PC-8800.md [hunk 7]
```diff
@@ -44,40 +25,171 @@ At this link you can find information on the BIOS files of the emulator:
...
+You can also set the q88_sub_cpu_mode in /opt/retropie/configs/all/retroarch-core-options.cfg :
+```
+q88_sub_cpu_mode = "2"
+```
+
+[Final Crisis] and many other games require a different CPU clock.
+Final Crisis works best on "8 Mhz".
+You can set this from the RetroArch menu :
```
Comment (cmitu) [inline docs/PC-8800.md:181]: _RetroArch menu_ should be renamed to _RetroArch core options_ - add a link to [RetroArch-Core-Options]

File: docs/PC-8800.md [hunk 8]
```diff
@@ -44,40 +25,171 @@ At this link you can find information on the BIOS files of the emulator:
...
+If you've loaded multiple disks, you can hold one of the shoulder buttons and use the D-Pad(Arrow-keys) to change the disk that's loaded in each drive. 
+Use L1-shoulder(Q-key) for Drive 1 and R1-shoulder(W-key) for Drive 2. 
+When the shoulder button is released, the chosen disk will be inserted.
+
+## Settings for games [lr-quasi88]:
+[Fire Hawk] and many other games only work if the sub_cpu_mode is always using both CPUs.
+We always kept this setting on, seems that emulation will always benefit.
+Earlier binary emulator versions didn't have this core option.
+So if it doesn't exist in the core options, you have to install the emulator from source.
+You can set this from the RetroArch menu :
```
Comment (cmitu) [inline docs/PC-8800.md:166]: RetroArch menu should be renamed to RetroArch core options - add a link to [RetroArch-Core-Options]

**Comment by cmitu** [comment]
Thank you for the contribuions. I added some comments on specific lines in the document that I think should be changed.

Some general comments on the modifications:

 * don't add `[..]` to headings, use sub-headings to organize the information, i.e.
```
## Emulator: Quasi88
...
### BIOS
### Controls
### Game tweaks.
...
```
* You've added a few new BIOS files, but no checksum for them. If they're documented somewhere - with the correct checksum - it would be good to have the checksum added.
* Try not to leave too many blank lines. If you want to add a line break (i.e. start a new paragraph), just add 4 spaces at the  end of the line.

**Comment by FollyMaddy** [comment]
Thank you for the comments.
I will try to work on your recommendations.

File: docs/PC-8800.md (hunk 2 above)
Comment (FollyMaddy) [reply docs/PC-8800.md:54]: Quasi88 works with uppercase files. Lr-quasi88 looks first for uppercase files then lowercase files. Discovered this in while reading the lr-quasi88 logs.

**Comment by cmitu** [comment]
> Do I have to attach the file in the conversation again ?

Modify the file (using the same git branch) and push to your repository, the modifications should show up in this PR. There's no need for a PR. If you modify the sections I commented, the modifications will appear there and will be easier to continue/resolve the review.

**Comment by FollyMaddy** [comment]
Sorry, I have to close this pull request.
I don't have enough information, knowledge and time to finish this right now.

**Comment by cmitu** [comment]
Let's leave the PR open, I'll try to re-format the info your added.

**Comment by FollyMaddy** [comment]
Ok, that would be really nice.

@zering has also added md5sums in the thread.
https://retropie.org.uk/forum/topic/28345/old-computer-appreciation-thread/1022
It would be nice if this could be implemented too, somehow.

**Comment by cmitu** [comment]
@FollyMaddy I have updated the PC88 page with your suggestions and some other modifications in #64. If you'd like to take a look, maybe there's something to new to add that I have overlooked.

**Comment by FollyMaddy** [comment]
@mitu 
Thank you for the update.
If I have some time I will look at it.
I will probably ask @zering also, in the old computer thread, to take a look at it.
I will let you know if we find something that has to be added / changed.
